app/src/main/resources/hikariJdbc.properties

```diff
@@ -2,7 +2,7 @@
 
 jdbc.url=jdbc:postgresql://localhost:5432/postgres
 jdbc.username=postgres
-jdbc.password=
+jdbc.password=testpassword
```

Comment:
Are users expected to change this? If so, could we name it such that this is obvious? Isn't clear to me if this has something to do with testing.

Reply:
How about "YOU_MUST_CHANGE_THIS_PASSWORD" or something as obvious?